WIFEXITED definitions: POSIX documents this in stdlib.h as optional, and is
defined there on linux, but on FreeBSD it is not, so we need to include
sys/wait.h where POSIX mandates it to be.


--- webkitgtk-1.10.2/Source/WebKit2/UIProcess/Plugins/gtk/PluginProcessProxyGtk.cpp.old	2013-01-28 20:38:45.000000000 -0300
+++ webkitgtk-1.10.2/Source/WebKit2/UIProcess/Plugins/gtk/PluginProcessProxyGtk.cpp	2013-01-28 20:39:10.000000000 -0300
@@ -35,6 +35,7 @@
 #include <glib.h>
 #include <wtf/text/CString.h>
 #include <wtf/text/WTFString.h>
+#include <sys/wait.h> // WIFEXITED
 
 using namespace WebCore;
 
--- webkitgtk-1.10.2/Source/WebKit2/UIProcess/Launcher/gtk/ProcessLauncherGtk.cpp.old	2013-01-28 20:40:26.000000000 -0300
+++ webkitgtk-1.10.2/Source/WebKit2/UIProcess/Launcher/gtk/ProcessLauncherGtk.cpp	2013-01-28 20:40:46.000000000 -0300
@@ -38,6 +38,7 @@
 #include <wtf/text/WTFString.h>
 #include <wtf/gobject/GOwnPtr.h>
 #include <wtf/gobject/GlibUtilities.h>
+#include <sys/wait.h> // WIFEXITED
 
 #if OS(LINUX)
 #include <sys/prctl.h>
